anesthetic/plot.py

```diff
@@ -768,6 +814,8 @@ def fastkde_plot_1d(ax, data, *args, **kwargs):
     i = ((x > quantile(x, q[0], p)) & (x < quantile(x, q[-1], p)))
 
     area = np.trapz(x=x[i], y=p[i]) if density else 1
+    if ax.get_xaxis().get_scale() == 'log':
+        x = 10**x
```

Comment:
We could avoid this hard-coding by using the [existing matplotlib functionality](https://matplotlib.org/stable/api/scale_api.html#matplotlib.scale.LogScale.get_transform)

Reply:
This is probably overkill until/if we implement symlog/logit, so leave it as-is for now.